mlflow/utils/requirements_utils.py

```diff
@@ -281,6 +281,10 @@ def _capture_imported_modules(model_uri, flavor):
             return f.read().splitlines()
 
 
+DATABRICKS_MODULES_TO_PACKAGES = {
+    "databricks.automl": ["databricks-automl-runtime"],
+    "databricks.model_monitoring": ["databricks-model-monitoring"],
```

Comment:
Can Model Serving serve a model that depends on `databricks-model-monitoring` (which is not available on PyPI)?

Reply:
Usually, a model doesn't depend on `databricks.model_monitoring`, right?

Reply:
Correct, a model doesn't usually depend on this, but Serving folks are adding support for it just in case.